src/connectors/voice-search/connectVoiceSearch.ts

```diff
@@ -117,6 +117,7 @@ const connectVoiceSearch: VoiceSearchConnector = (
       },
       dispose({ state }) {
         unmountFn();
+        (this as any)._voiceSearchHelper.removeEventListeners();
```

Comment:
We should also stop listening once we dispose the widget.

Reply:
https://github.com/algolia/instantsearch.js/pull/3779/commits/d95a9bcaf0f77b357b5b649272aa96ef193297f4